domains/pele.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Pele",
+    "email": "pele.puric@gmail.com",
+    "discord": "483261654818226186"
+  },
+  "record": {
+    "A": "http://38.242.242.224/"
```

Comment:
A can only contain IP, not HTTP links
```suggestion
    "A": "38.242.242.224"
```
https://is-a.dev/docs/domain-structure

Reply:
Also provide a site preview, the maintainer prob ask you about that
![image](https://github.com/user-attachments/assets/70ac0ac2-a625-41d6-8a96-2594639997bf)

Reply:
this, btw if he wants to use links, he can use URL records

Reply:
> this (cupglass' review), btw if he wants to use links, he can use URL records

hmm thats a good reason also